settings/build.gradle

```diff
@@ -34,16 +34,17 @@ android {
     kotlinOptions {
         jvmTarget = '1.8'
     }
-
     packagingOptions {
-        // These library licenses will be referenced in-app
-        exclude 'META-INF/ASL-2.0.txt'
-        exclude 'META-INF/LGPL-3.0.txt'
+        resources {
```

Comment:
There's been a small tweak to the `packagingOptions` block - we now have `exclude`s etc in the `resources` block.